app.rb

```diff
@@ -0,0 +1,110 @@
+require_relative 'lib/database_connection'
+require_relative './lib/item_repository'
+require_relative './lib/order_repository'
+
+class Application
+    def initialize(database_name, io, item_repository, order_repository)
+        DatabaseConnection.connect(database_name)
+        @io = io
+        @item_repository = item_repository
+        @order_repository = order_repository
+    end
+
+    def run
+        @io.puts "Welcome to the shop management program!"
+        @io.puts "What would you like to do?"
+        @io.puts "1 = List all shop items"
+        @io.puts "2 = Create a new item"
+        @io.puts "3 = List all orders"
+        @io.puts "4 = Create a new order"
+        @io.puts "5 = Increase stock of item"
+        @io.puts "6 = Delete an order"
+        get_choice
+    end
+
+
+    private 
+
+    def get_choice 
+        input = @io.gets.chomp
+        fail "Please ensure input is either one of 1, 2, 3, 4, 5 or 6" unless input.to_i == 1 || input.to_i == 2 || input.to_i == 3 || input.to_i == 4 ||  input.to_i == 5 || input.to_i == 6
```

Comment:
This line is doing a lot of lifting! Could this be a separate private method, to enable the code to flow more nicely?